pkg-config.cmake

```diff
@@ -892,8 +899,15 @@ ENDMACRO(BUILD_PREFIX_FOR_PKG)
 #       PKG_CONFIG_USE_DEPENDENCY(my-binary my-package)
 #
 MACRO(PKG_CONFIG_USE_DEPENDENCY TARGET DEPENDENCY)
+  SET(options OPTIONAL NO_INCLUDE_SYSTEM)
```

Comment:
I think OPTIONAL is not expected here. It is understood by cmake_parse_arguments as an argument like any others.